web/app_dev.php

```diff
@@ -17,14 +17,20 @@
     exit('You are not allowed to access this file. Check '.basename(__FILE__).' for more information.');
 }
 
-$loader = require_once __DIR__.'/../app/bootstrap.php.cache';
+$loader = require_once dirname($_SERVER['SCRIPT_FILENAME']).'/../app/bootstrap.php.cache';
 Debug::enable();
 
-require_once __DIR__.'/../app/AppKernel.php';
+require_once dirname($_SERVER['SCRIPT_FILENAME']).'/opt/akeneo/app/AppKernel.php';
```

Comment:
Is this right? This is the only one with /opt/akeneo

Reply:
It's likely wrong and I don't think we've ever used `app_dev.php`. These files will be scrapped completely when we upgrade akeneo, so I'm OK with not fixing it.